lib/instrumentation/modules/aws-sdk.js

```diff
@@ -0,0 +1,40 @@
+'use strict'
+const semver = require('semver')
+const shimmer = require('../shimmer')
+const { instrumentationSqs } = require('./aws-sdk/sqs')
+
+// Called in place of AWS.Request.send
```

Comment:
nit: and .promise now as well.

Reply:
✅ https://github.com/elastic/apm-agent-nodejs/commit/d99a68f64fc4b61a7948966da03107448bd6255d